### ISSUE TITLE
Could you please provide the Multicue dataset after data augmentation？

### ISSUE THREAD

**Issue by zhoujh2020** [MengyangPu/EDTER#33]
I am really interested to your work. Could you please provide the Multicue dataset after data augmentation to me for reproducing the results？I will be very grateful for your help.

**Comment by MengyangPu** [comment]
we augment the training set by random horizontal flipping, rotating (90, 180, and 270◦), and scaling (75 and 125%) images and their corresponding ground-truth label maps.

**Comment by zhoujh2020** [comment]
Sorry to bother you again. Would you like to preprocess your data set first and then get a flip(2x), rotate(4x), scale(3x) data set, or do random horizontal flip, random rotation and random scaling of a picture obtained in the dataloader through code? Thank you very much for your help, looking forward to your answer.

**Comment by MengyangPu** [comment]
I uploaded the augmentation data, [link](https://pan.baidu.com/s/1HKIumIcX0qgvVR_1xvWInA?pwd=gw2r) (code: gw2r).
